spec/post_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+  end
+
+  context 'Validations' do
+    it 'checks for title presence' do
+      post = Post.new(author: @user, text: 'This is my first post', likes_counter: 0, comments_counter: 0)
+      expect(post.valid?).to eq false
+    end
+
```

Comment:
- [ ] Please fix this error as requested by the previous reviewer.

![image](https://user-images.githubusercontent.com/83803414/200135205-1c8289bc-c88c-4219-b233-3fda18fe4044.png)